engine-tests/src/test/java/org/terasology/engine/persistence/internal/StorageManagerTest.java

```diff
@@ -100,13 +101,10 @@ static void createFolder() throws IOException {
     }
 
     @BeforeEach
-    public void setup(@TempDir Path tempHome) throws Exception {
+    public void setup() throws Exception {
         super.setup();
-        PathManager.getInstance().useOverrideHomePath(tempHome);
         savePath = PathManager.getInstance().getSavePath("testSave");
 
-        assert !Files.isRegularFile(tempHome.resolve("global.dat"));
```

Comment:
why is this assertion no longer relevant?

Reply:
this should use the junit test function not assert.

Reply:
I think the intention of that assert was to make sure that we aren't re-using a directory that already has stuff in it. I took it out because it was already guaranteed unique by `@TempDir`, and because I removed this TempDir stuff because there's already TempDir stuff setting up the PathManager in TerasologyTestingEnvironment.

But your comment did prompt me to take another look, and I found that I had broken things here, I just hadn't gotten caught yet. (`global.dat` would be under `savePath`, not `tempHome`) This is a `BeforeEach`, but TTE was only doing the path setup `BeforeAll`.

I've fixed it now so every test gets its own save name, regardless of whether the PathManager has been reset between methods or not.